packages/components/src/time-since/index.tsx

```diff
@@ -88,6 +88,13 @@ function formatDate( date: Date, format: string, locale?: string ): string {
 		// Medium date format without time
 		formatOptions.dateStyle = 'medium';
 		delete formatOptions.timeStyle;
+	} else if ( format === 'lll' ) {
```

Comment:
This component would be a good candidate for test coverage to ensure the date is being formatted as we expect it to be.